client/js/socket-events/join.js

```diff
@@ -10,11 +10,14 @@ const sidebar = $("#sidebar");
 socket.on("join", function(data) {
 	const id = data.network;
 	const network = sidebar.find(`#network-${id}-chanlist`);
-	network.append(
-		templates.chan({
-			channels: [data.chan],
-		})
-	);
+
+	const channels = network.children();
```

Comment:
This seems a bit brittle, for example if we had an extra container or something. Is it worth looking for something a bit more specific, like `.channel` or something? I don't know

Reply:
Probably. It's following the same code that we use in the sync_sort though, so we should probably change both of them. Can we do that in another PR?

Reply:
Up to you and @xPaw. I'd rather not check in something we want to change directly, but ultimately I don't really mind.

Reply:
@McInkay willing to do a follow up PR?